app/src/main/java/io/github/iurimenin/upcomingmovies/model/MovieVO.kt

```diff
@@ -8,23 +8,38 @@ import java.util.*
 /**
  * Created by Iuri Menin on 26/08/17.
  */
-class MovieVO : Parcelable {
+class MovieVO protected constructor(`in`: Parcel) : Parcelable {
```

Comment:
Why the model classes implements `Parcelable`?

Reply:
Because I send the object from one activity to another by intent